pydy/viz/scene.py

```diff
@@ -459,133 +458,147 @@ def remove_static_html(self, force=False):
             print "aborted!"
 
     def display(self):
-        """
-        display method can be used in two ways.
-        When called from IPython notebook, it shows the visualization
-        in the form of output cell in the IPython notebook.
-        If it is called from python interpreter or
-        IPython interpreter(not notebook), It generates an html file,
-        in the current directory, which can be opened in the webgl
-        compliant browser for viewing the visualizations.
-
-        The simulation data is used from this scene, hence
-        all simulation data generation methods should be called before
-        calling this method
-
-        """
+        """Displays the scene in the default webbrowser."""
         self.create_static_html()
-        run_server(scene_file=self.scene_json_file)
+        run_server(scene_file=self._scene_json_file)
 
-    def display_ipython(self):
-        """
-        Method to display the visualization inside the
-        IPython notebook. It is only supported by IPython
-        versions>=2.0.0
+    def _rerun_button_callback(self, clicked):
```

Comment:
This (https://ipython.org/ipython-doc/dev/api/generated/IPython.html.widgets.widget_button.html) says:

"The callback will be called with one argument, the clicked button widget instance."